### PR TITLE
Rename everything to skippy and center skippy

### PR DESCRIPTION
**Pull request by MartijnCuppens**
Both _skippy_ and _skiplink_ are used to name skippy in the docs. ~~This PR renames everything to skiplink and centers the skiplink.~~ This PR renames everything to skippy and centers skippy.

### CODE REVIEW

File: site/docs/4.1/assets/scss/_skiplink.scss
Comment (XhmikosR) [inline site/docs/4.1/assets/scss/_skiplink.scss]: We could probably use our utils here more. Not sure what mdo prefers though.

**Comment by XhmikosR** [comment]
@MartijnCuppens Did you verify the ID isn't actually used anywhere?

File: site/docs/4.1/assets/scss/_skiplink.scss
Comment (MartijnCuppens) [reply site/docs/4.1/assets/scss/_skiplink.scss]: It would be a long list of utility classes and in the and we'll still need to add additional css for `background-color` & `outline`, that's why I preferred to keep this here.

**Comment by MartijnCuppens** [comment]
@XhmikosR, I searched for all occurrences of `skippy` and replaced them.

**Comment by XhmikosR** [comment]
OK, I will verify later too.

**Comment by XhmikosR** [comment]
OK, LGTM. One thing. While at it how about changing the `:hover:focus` color to white?

**Comment by MartijnCuppens** [comment]
Updated the hover color. (Focus color doesn't need to be set).

**Review by XhmikosR — APPROVED** [review]

**Comment by XhmikosR** [comment]
LGTM. Waiting for @mdo's approval.

**Comment by patrickhlauke** [comment]
RIP skippy 
250px-skippy-dvd

**Comment by XhmikosR** [comment]
haha

**Comment by XhmikosR** [comment]
BTW I don't mind keeping "skippy".

**Comment by MartijnCuppens** [comment]
We can still save skippy, the issue was that there was just an inconsistent naming of skippy/skiplink.

**Comment by MartijnCuppens** [comment]
Skippy is back with us ❤️

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**
